### ISSUE TITLE
Invalid python package for django

### ISSUE THREAD

**Issue by progremir** [highlight/highlight#6176]
**Describe the bug**
In the docs you specify a non-existent package/bundle for django.

**To Reproduce**

1. Go to https://www.highlight.io/docs/getting-started/backend-sdk/python/django
3. Scroll down to "Install the highlight-io python package."

**Expected behavior**
Just show `pip install highlight-io` or publish a `highlight-io[Django]` bundle

**Screenshots**
<img width="876" alt="image" src="https://github.com/highlight/highlight/assets/17231674/45ea829a-cbd5-485a-b559-de7aa1e9e8b7">
<img width="615" alt="image" src="https://github.com/highlight/highlight/assets/17231674/b0c44581-8c3f-4790-8f8d-f39ab0c7058e">

**Additional context**

**Comment by Vadman97** [comment]
Thanks @progremir , will be fixing this week!

**Comment by Vadman97** [comment]
@progremir we ship `highlight-io` with a flavor `Django` that can be installed as shown.
Closing as I do not believe this is an issue; please let me know otherwise.
```bash
(venv)  py[venv]  ~  pip install highlight-io[Django]
Collecting highlight-io[Django]
  Downloading highlight_io-0.6.0-py3-none-any.whl (10 kB)
...
Installing collected packages: zipp, wrapt, urllib3, typing-extensions, sqlparse, protobuf, opentelemetry-semantic-conventions, idna, grpcio, charset-normalizer, certifi, backoff, requests, opentelemetry-proto, importlib-metadata, googleapis-common-protos, deprecated, asgiref, opentelemetry-exporter-otlp-proto-common, opentelemetry-api, django, opentelemetry-sdk, opentelemetry-instrumentation, opentelemetry-instrumentation-logging, opentelemetry-exporter-otlp-proto-http, opentelemetry-exporter-otlp-proto-grpc, opentelemetry-distro, opentelemetry-exporter-otlp, highlight-io
Successfully installed asgiref-3.7.2 backoff-2.2.1 certifi-2023.7.22 charset-normalizer-3.2.0 deprecated-1.2.14 django-4.2.4 googleapis-common-protos-1.60.0 grpcio-1.57.0 highlight-io-0.6.0 idna-3.4 importlib-metadata-6.8.0 opentelemetry-api-1.19.0 opentelemetry-distro-0.40b0 opentelemetry-exporter-otlp-1.19.0 opentelemetry-exporter-otlp-proto-common-1.19.0 opentelemetry-exporter-otlp-proto-grpc-1.19.0 opentelemetry-exporter-otlp-proto-http-1.19.0 opentelemetry-instrumentation-0.40b0 opentelemetry-instrumentation-logging-0.40b0 opentelemetry-proto-1.19.0 opentelemetry-sdk-1.19.0 opentelemetry-semantic-conventions-0.40b0 protobuf-4.24.1 requests-2.31.0 sqlparse-0.4.4 typing-extensions-4.7.1 urllib3-2.0.4 wrapt-1.15.0 zipp-3.16.2

[notice] A new release of pip is available: 23.1.1 -> 23.2.1
[notice] To update, run: pip install --upgrade pip

```

**Comment by progremir** [comment]
Thanks @Vadman97 !
